app/Http/Controllers/Admin/BatchController.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use Illuminate\Http\Request;
+use App\Http\Controllers\Controller;
+use App\Repositories\BatchRepository;
+use App\Repositories\WorkspaceRepository;
+use App\Repositories\PositionRepository;
+use App\Repositories\ProgramRepository;
+use App\Http\Requests\BatchRequest;
+
+class BatchController extends Controller
+{
+    protected $batch;
+    
+    public function __construct(
+        BatchRepository $batch,
+        ProgramRepository $programRepository,
+        PositionRepository $positionRepository,
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.NewlineBeforeOpenBrace</br>**Severity**: error</br>**File**: app/Http/Controllers/Admin/BatchController.php</br>**Line**: 22</br>The closing parenthesis and the opening brace of a multi-line function declaration must be on the same line